Controller/NotifyHandler.php

```diff
@@ -197,6 +197,10 @@ private function _onNotificationSuccess($oResponse, $oBackendService, $sPaymentI
         $oOrder = oxNew(Order::class);
         if ($this->_loadOrder($oOrder, $sTransactionId) >= self::MAX_TIMEOUT_SECONDS) {
             $this->_oLogger->error('No order found for transactionId: ' . $sTransactionId);
+            if ($oResponse->getPaymentMethod() === Transaction::WIRETRANSFER) { // unmatched POI/PIA transaction
+                // Unmatched transaction is saved only if it was a POI/PIA transaction
```

Comment:
Please move this comment above the whole if-block and also add an empty line before while removing the then redundant "unmatched POI/PIA transaction" comment.

Reply:
👍